wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
Okay, I made this PR just have the bug fix, since that's what we can address right away.

If/when you get access to the Pypi release, it might be wise to just bump the version number so downstream projects get the bug fixes. ;)